src/widgets/reflow.rs

```diff
@@ -124,16 +125,19 @@ pub struct LineTruncator<'a, 'b> {
     symbols: &'b mut dyn Iterator<Item = Styled<'a>>,
     max_line_width: u16,
     current_line: Vec<Styled<'a>>,
+    cut: u16,
```

Comment:
I'm not a big fan of the name. What do think of `horizontal_offset` ? Could you also add a doc comment to explain how is this property will be used when the `LineTrucator` process lines ?

Reply:
I agree with you, and will add a doc to explain.